### DISCUSSION TITLE
Mesh format

### DISCUSSION THREAD

**Discussion by jjywow** [su2code/SU2#1857] [Q&A] [answered]
Hello, 

I'm doing your tutorial 'structural mechanics with vertical cantilever.  I have question about mesh format.

Considering example as below, I understand that NELEM refers to total number of element.

second to fourth column refer to mesh connection. last column refers to element number.

What does the first column mean?

"NDIME= 2
NELEM= 1000
9 3 120 220 121 0
9 120 119 221 220 1
9 119 118 222 221 2"

Also for Marker , what does the first column means?

MARKER_TAG= clamped
MARKER_ELEMS= 10
3 0 4 
3 4 5 
3 5 6 
3 6 7 
3 7 8 
3 8 9 
3 9 10 
3 10 11 
3 11 12 
3 12 1 

Thanks

**Comment by pcarruscag** [comment, marked as answer]
It's the type of element
https://su2code.github.io/docs_v7/Mesh-File/

**Reply by jjywow** [reply]
Thanks a lot! I missed the page.
